label_studio/data_manager/api.py

```diff
@@ -224,11 +225,23 @@ def sync_paginate_queryset(self, queryset, request, view=None):
         self.total_annotations = Annotation.objects.filter(task_id__in=queryset, was_cancelled=False).count()
         return super().paginate_queryset(queryset, request, view)
 
+    def paginate_totals_queryset(self, queryset, request, view=None):
+        totals = queryset.values('id', 'total_predictions', 'total_annotations', 'cancelled_annotations').aggregate(
```

Comment:
Is `total_predictions`, `total_annotations` and `cancelled_annotations` needed in `values` if we are aggregating them afterwards?

Reply:
If not present in values the SELECT was missing these fields and failed to aggregate, I can try it again though just in case I missed something.

Reply:
Took it out, only need the id for the `SELECT DISTINCT`